build/scripts/CertSignAndInstall.ps1

```diff
@@ -9,7 +9,7 @@ function Invoke-SignPackage([string]$Path) {
         return
     }
 
-    $certName = "Microsoft.GITServices"
+    $certName = "Microsoft.GitHubExtension"
```

Comment:
Do we need to add this file or is it coming from the pipeline?

Reply:
This is generated for you when building locally